docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+- If you won't always use a ground station to set MAVLink stream rates when it starts up, be sure to set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.
+
+- See the [Yaapu Wiki](https://github.com/yaapu/FrskyTelemetryScript/wiki) and [mLRS CRSF Telemetry](CRSF.md) page for details.
+
+## 7. Connect a ground station to your mLRS Tx module
+
+There are MAVLink ground station applications available for all common platforms including Windows, MacOS, Linux, Android, and iOS. We won't cover selecting a platform or installing a ground station here.  But we will provide some hints and suggestions for popular connection methods.
+
+Wireless links are possible for all three of the recommended Tx modules.
+
+Bluetooth is very handy for mobile devices and is recommended for the MatekSys Tx modules since they include an HC04 module which is automatically configured by mLRS.  Bluetooth is also supported by the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) which is recommended for use with the [FrSky R9M Tx module](FRSKY_R9.md#esp32-wireless-bridge).
+- The [HC04](MATEKSYS.md#tx-module-hc-04-bluetooth-notes) will be discoverable soon after being powered on.
+- First, pair your HC04 (pin: 1234) or esp32 with your device or computer.  Look for a device which includes "mLRS" in its name.
+- iOS devices do not support the Bluetooth Serial Port Profile and QGroundControl does not support Bluetooth LE so QGroundControl will not connect over Bluetooth on iOS.  It has been reported that the SidePilot App works on iOS.
+- QGroundControl works well with Bluetooth on Android, Linux or Windows.  Create a new Bluetooth type connection via Application Settings -> Comm Links -> Add.
+- Mission Planner is recommended for Windows.  Connect to the com port associated with the Bluetooth connection after pairing.
+
+UDP or TCP over WiFi is another popular wireless option.
+- All three of the recommended Tx modules could be connected to an external esp32 board such as the M5 Stamp boards running the [Wireless Bridge](WIRELESS_BRIDGE.md) sketch.
+- All popular ground stations can connect via UDP
+- The standard UDP port for MAVLink is 14550.
+
+All three of the recommended Tx modules include serial ports which may be used to connect to your ground station.
+- Since PC serial ports are full RS232 ports which use higher voltage levels than the logic level serial connections on Tx modules, a USB ttl serial adapter is used to connect a Tx serial port to a PC or mobile device.
+- The FrSky R9M shares its single inverted serial port between CLI and MAVLink based on the position of [dip switch 1](FRSKY_R9.md#r9m-tx-module) at power on.  A [method](FRSKY_R9.md#diy-inverter-dongle) to invert the signals will be required.
+- The [MatekSys Tx modules](https://www.mateksys.com/?portfolio=mr24-30-tx#tab-id-2) provide 1 or 2 TTL level serial ports depending on the dip switch settings.
```

Comment:
I think this bullet should come before the FrSky

- matches the pattern 
- better to go from simple to complicated